lib/Frontend/nvqpp/ConvertExpr.cpp

```diff
@@ -2168,6 +2183,24 @@ bool QuakeBridgeVisitor::VisitCXXOperatorCallExpr(
       auto idx_var = popValue();
       auto qreg_var = popValue();
 
+      // We could have the case where the qvector is coming from a struct
+      // and is not in the symbol table.
```

Comment:
Why wouldn't this be an expression like `var.qview_member`?

Reply:
I would have thought so. When I ran the code as is (`h(var.qview_member[2]);`, ), the bridge threw an error here - it was looking for the variable name (`qview_member`) in they symbol table.

Reply:
That's, well, weird. Need to figure that out. (I thought members were working fine.)